app/controllers/api/v1/plans_controller.rb

```diff
@@ -17,7 +17,13 @@ def create
 
       def update
         plan = current_organization.plans.parents.find_by(code: params[:code])
-        result = ::Plans::UpdateService.call(plan:, params: input_params.to_h.deep_symbolize_keys)
+
+        update_params = input_params.to_h.deep_symbolize_keys
+        if params[:plan].key?(:cascade_updates)
+          update_params[:cascade_updates] = params.require(:plan).permit(:cascade_updates)[:cascade_updates]
+        end
```

Comment:
What's the reason for handling this at this level rather than in the `input_params`?

Reply:
This param is needed only for `update` action. Another option was to have some logic in `input_params` method...